.github/workflows/update.yml

```diff
@@ -18,3 +18,4 @@ jobs:
         with:
           inputs: nixpkgs
           pr-title: Update systems
+          token: ${{ secrets.GH_TOKEN_FOR_UPDATES }}
```

Comment:
I can't view settings so i can't verify that the token is correct, but i trust you configured it correctly

Reply:
It is correct i tested it by running the updater manually on this branch